packages/mobile/src/utils/formatting.ts

```diff
@@ -32,16 +32,47 @@ export const getFeeDisplayValue = (value: BigNumber.Value | null | undefined): s
   return value ? numeral(BigNumber.max(value, 0.001).toNumber()).format('0[.][0000]') : ''
 }
 
+/**
+ * More precise getFeeDisplayValue with built in rounding
+ * Used for small Network Fees in transaction feed
+ * @param value fee amount
+ */
+export const getNetworkFeeDisplayValue = (value: BigNumber.Value): string => {
```

Comment:
Could we just add a param to `getFeeDisplayValue` instead of a new util function that's only used once?